src/Sylius/Bundle/LocaleBundle/Templating/Helper/LocaleHelper.php

```diff
@@ -27,19 +27,30 @@
     private $localeConverter;
 
     /**
+     * @var string
+     */
+    private $defaultLocale;
+
+    /**
      * @param LocaleConverterInterface $localeConverter
+     * @param string $defaultLocale
      */
-    public function __construct(LocaleConverterInterface $localeConverter)
+    public function __construct(LocaleConverterInterface $localeConverter, string $defaultLocale)
```

Comment:
BC promise is still a work in progress, but it looks like a BC break (adding required parameters to the public method).

Reply:
So, should I make the defaultLocale parameter null by default (doing ?string $defaultLocale = null) ?